task/04-arrays-tasks.js

```diff
@@ -304,7 +333,16 @@ function get3TopItems(arr) {
  *   [ 1, '2' ] => 1
  */
 function getPositivesCount(arr) {
-   throw new Error('Not implemented');
+   let count = 0;
+   arr.map(function(element) {
+      if (typeof element == "number" && element > 0)
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
'0' == 0
and '0' === 0)"